examples/jsonnet/accessPolicies.jsonnet

```
@@ -0,0 +1,16 @@
+local argokit = import '../../../argokit/jsonnet/argokit.libsonnet';
+
+argokit.Application('testapp')
++argokit.AccessPolicies.inbound.skipApp('app')
++argokit.AccessPolicies.inbound.skipApp('otherApp', 'mynamespace')
++argokit.AccessPolicies.inbound.skipAppByTeam('otherApp', 'team')
++argokit.AccessPolicies.outbound.skipAppJob('otherApp')
++argokit.AccessPolicies.outbound.skipAppByTeam('teamApp', 'team')
++argokit.AccessPolicies.outbound.skipAppByTeamJob('teamAppJob', 'team')
++argokit.AccessPolicies.outbound.postgres('postgres', '192.168.1.1')
++argokit.AccessPolicies.outbound.oracle('oracle', '192.168.1.2')
++argokit.AccessPolicies.outbound.ssh('ssh', '192.168.1.3')
++argokit.AccessPolicies.outbound.ldaps('ldaps', '192.168.1.4')
++argokit.AccessPolicies.outbound.ldap('ldap', '192.168.1.5')
++argokit.AccessPolicies.outbound.http('http')
++argokit.AccessPolicies.outbound.httpJob('httpJob')
```

Ville brukt noen hostnames her, f.eks.
```suggestion
+argokit.AccessPolicies.outbound.http('cache.kartverket.no')
+argokit.AccessPolicies.outbound.httpJob('storage.googleapis.com')
```